narwhals/series.py

```diff
@@ -472,6 +487,9 @@ def dtype(self: Self) -> DType:
     def name(self) -> str:
         """Get the name of the Series.
 
+        Returns:
+            The name of the Series as a string.
```

Comment:
```suggestion
            The name of the Series.
```
we allow non-string names (even though we don't encourage them 😄 )